lib/wallaby.ex

```diff
@@ -46,9 +46,13 @@ defmodule Wallaby do
   end
 
   def js_errors? do
-    Application.get_env(:wallaby, :js_errors) || true
+    Application.get_env(:wallaby, :js_errors)
```

Comment:
`Application.get_env/3` actually accepts a default (which is what I should have used in the first place). Thanks for catching my silly mistake.